django/core/checks/security/base.py

```diff
@@ -68,8 +68,9 @@
 
 W009 = Warning(
     "Your SECRET_KEY has less than %(min_length)s characters or less than "
-    "%(min_unique_chars)s unique characters. Please generate a long and random "
-    "SECRET_KEY, otherwise many of Django's security-critical features will be "
+    "%(min_unique_chars)s unique characters, or it's django-generated secret key starting with dj::insecure."
```

Comment:
`it's django-generated` --> `it's a Django-generated`

Reply:
Yes, I sometimes failing with English, thank you!